src/calculating.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const calculating = (number, answer) => {
+  const bulls = [];
+  const cows = [];
+
+  const len = answer.toString().length;
+
+  for (let i = 0; i < len; i++) {
+    const pcValue = number.toString().split('')[i];
+    const inputValue = answer.toString().split('')[i];
```

Comment:
If you need to do some operations on the data in the middle of the loop, it's better to move it outside the loop, because now `number.toString().split('')` and `answer.toString().split('')` will be executed on each iteration of the loop (that is, four times), although it is enough to do it once before starting the cycle. This change will improve the performance of the code.